src/blocks/blocks/popup/save.js

```diff
@@ -36,7 +36,7 @@ const Save = ({
 				<div className="otter-popup__modal_content">
 					{ attributes.showClose && (
 						<div className="otter-popup__modal_header">
-							<button type="button" class="components-button has-icon"><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img" aria-hidden="true"><path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></path></svg></button>
+							<button type="button" className="components-button has-icon"><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" role="img" aria-hidden="true"><path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></path></svg></button>
```

Comment:
This is frontend HTML, it should not be using className but class as attribute.

Reply:
Upon reviewing this more, I think this is fine. I'll just double check this to confirm it doesn't cause any invalidation with the change in save.js.